code/game/objects/items/stacks/tiles/tile_types.dm

```diff
@@ -20,25 +20,14 @@
 	drop_sound = 'sound/items/drop/axe.ogg'
 	pickup_sound = 'sound/items/pickup/axe.ogg'
 
-//crafting / welding vars
-	var/datum/material/material //*sigh* i guess this is how we're doing this.
-	var/craftable = FALSE //set to TRUE for tiles you can craft stuff from directly, like grass
+//welding vars
 	var/can_weld = FALSE //set to TRUE for tiles you can reforge into their components via welding, like metal
 	var/welds_into = /obj/item/stack/material/steel //what you get from the welding. defaults to steel.
-	var/default_type = DEFAULT_WALL_MATERIAL

```

Comment:
?

Reply:
this fixes dismantling protolathes giving grass tiles, while keeping the weld-to-reforge functionality
~~aka fixes a really weird bug i introduced in #9002 unrelated to the morphium thing which apparently has been a thing for ages~~